.github/workflows/pr_build.yaml

```diff
@@ -208,16 +212,56 @@ jobs:
           key: ${{ runner.os }}-tools-${{ github.sha }}
           restore-keys: |
             ${{ runner.os }}-tools-
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
       - name: Build scratch images
         run: make scratch-images
       - name: Export scratch images
-        run: docker save spire-server-scratch:latest-local spire-agent-scratch:latest-local k8s-workload-registrar-scratch:latest-local oidc-discovery-provider-scratch:latest-local | gzip > scratch-images.tar.gz
+        run: tar -czvf scratch-images.tar.gz *-scratch-image.tar
       - name: Archive scratch images
         uses: actions/upload-artifact@v3
         with:
           name: scratch-images
           path: scratch-images.tar.gz
 
+  static-images:
```

Comment:
This workflow adds another variant of the docker image using chainguards wolfi image.